docs/install_csi_driver.md

```diff
@@ -0,0 +1,174 @@
+
+If your management cluster is already initialized, you can also enable the `ClusterResourceSet` by changing configuration of the  `capi-controller-manager` deployment in the `capi-system` namespace.
+
+```shell
+kubectl edit deployment -n capi-system capi-controller-manager
+```
+
+Just search the following part
+
+```yaml
+  - args:
+    - --leader-elect
+    - --metrics-bind-addr=localhost:8080
+    - --feature-gates=MachinePool=false,ClusterResourceSet=true,ClusterTopology=false
+```
+
+and replace `ClusterResourceSet=false` by `ClusterResourceSet=true`
+
+Editing the `deployment` resource will cause Kubernetes to automatically start new versions of the containers with the feature enabled.
+
+
+
+### Prepare Nutanix CSI ClusterResourceSet
+
+#### Create the ConfigMap for the CNI Plugin
+
+The first step is to create a `ConfigMap` that contains a YAML manifest with all resources to install the Nutanix CSI driver.
+
+As the Nutanix CSI is provided as Helm Charts, we will use `helm` to extract it before creating the `ConfigMap`.
+
+```shell
+helm repo add nutanix https://nutanix.github.io/helm/
+helm repo update
+
+kubectl create ns ntnx-system --dry-run=client -o yaml > nutanix-csi-namespace.yaml
+helm template nutanix-csi-snapshot nutanix/nutanix-csi-snapshot -n ntnx-system > nutanix-csi-snapshot.yaml
+helm template nutanix-csi-snapshot nutanix/nutanix-csi-storage -n ntnx-system > nutanix-csi-storage.yaml
+
+kubectl create configmap nutanix-csi-crs --from-file=nutanix-csi-namespace.yaml --from-file=nutanix-csi-snapshot.yaml --from-file=nutanix-csi-storage.yaml
+```
+
+#### Create the ClusterResourceSet
+
+Next you need to create the `ClusterResourceSet` resource that will map your `ConfigMap` defined above to clusters using `clusterSelector` of your choice.
+
+Please find an example below, this `ClusterResourceSet` needs to be created inside your management cluster.
+
+```yaml
+---
+apiVersion: addons.cluster.x-k8s.io/v1alpha3
+kind: ClusterResourceSet
+metadata:
+  name: nutanix-csi-crs
+spec:
+  clusterSelector:
+    matchLabels:
+      csi: nutanix 
+  resources:
+  - kind: ConfigMap
+    name: nutanix-csi-crs
+```
+
+The `clusterSelector` field controls how Cluster API will match this `ClusterResourceSet` on one or more workload clusters. In this case, I’m using a `matchLabels` approach where the `ClusterResourceSet` will be applied to all workload clusters having the `csi: nutanix` label present. If the label isn’t present, the `ClusterResourceSet` won’t apply to that workload cluster.
+
+The `resources` field references the ConfigMap created above, which contains the manifests for installing the Nutanix CSI driver.
+
+#### Assign the ClusterResourceSet to a workload cluster
+
+To assign this `ClusterResourceSet` to your workload cluster this is pretty simple, you just need to add the right label to your `Cluster` resource.
```

Comment:
"to your workload cluster ->this<- is pretty simple" => "to your workload cluster is pretty simple"